buildpack.md

```diff
@@ -91,6 +80,54 @@ Buildpack API versions:
  - `<major>` and `<minor>` MUST only contain numbers (unsigned 64 bit integer)
  - When `<major>` is greater than `0` increments to `<minor>` SHALL exclusively indicate additive changes
 
+## Terminology
+
+### CNB Terminology
+
+A **buildpack** is a directory containing a `buildpack.toml`.
```

Comment:
While technically true, we probably want a definition to also describe what a buildpack does?

```suggestion
A **buildpack** is a directory containing a `buildpack.toml`.
```

Reply:
Updated in ff54426